<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilemanageForm</class>
 <widget class="QWidget" name="FilemanageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72,61, 139);</string>
  </property>
  <widget class="QPushButton" name="editFileBut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/editFile.bmp);</string>
   </property>
   <property name="text">
    <string>编辑文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manageFileBut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/manageFile.bmp);</string>
   </property>
   <property name="text">
    <string>管理文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>144</width>
     <height>144</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/exit.bmp);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>1061</width>
     <height>761</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
